core/src/foundations/ionic.vars.scss

```
@@ -939,6 +367,578 @@ $ion-primitives-pink-1100: var(--ion-primitives-pink-1100, #8b2248);
+$ion-semantics-warning-600-rgb: var(--ion-semantics-warning-600-rgb, 249, 189, 10);
+$ion-semantics-warning-700: var(--ion-semantics-warning-700, $ion-primitives-yellow-700);
+$ion-semantics-warning-700-rgb: var(--ion-semantics-warning-700-rgb, 223, 156, 0);
+$ion-semantics-warning-800: var(--ion-semantics-warning-800, $ion-primitives-yellow-800);
+$ion-semantics-warning-800-rgb: var(--ion-semantics-warning-800-rgb, 186, 130, 0);
+$ion-semantics-warning-900: var(--ion-semantics-warning-900, $ion-primitives-yellow-900);
+$ion-semantics-warning-900-rgb: var(--ion-semantics-warning-900-rgb, 150, 101, 3);
+$ion-semantics-warning-1000: var(--ion-semantics-warning-1000, $ion-primitives-yellow-1000);
+$ion-semantics-warning-1000-rgb: var(--ion-semantics-warning-1000-rgb, 112, 75, 2);
+$ion-semantics-warning-1100: var(--ion-semantics-warning-1100, $ion-primitives-yellow-1100);
+$ion-semantics-warning-1100-rgb: var(--ion-semantics-warning-1100-rgb, 83, 56, 2);
+$ion-semantics-warning-1200: var(--ion-semantics-warning-1200, $ion-primitives-yellow-1200);
+$ion-semantics-warning-1200-rgb: var(--ion-semantics-warning-1200-rgb, 44, 30, 2);
+$ion-semantics-warning-base: var(--ion-semantics-warning-base, $ion-semantics-warning-500);
+$ion-semantics-warning-base-rgb: var(--ion-semantics-warning-base-rgb, 255, 214, 0);
+$ion-scale-0: var(--ion-scale-0, 0px);
+$ion-scale-100: var(--ion-scale-100, 4px);
+$ion-scale-150: var(--ion-scale-150, 6px);
+$ion-scale-200: var(--ion-scale-200, 8px);
+$ion-scale-250: var(--ion-scale-250, 10px);
+$ion-scale-300: var(--ion-scale-300, 12px);
+$ion-scale-400: var(--ion-scale-400, 16px);
+$ion-scale-500: var(--ion-scale-500, 20px);
+$ion-scale-600: var(--ion-scale-600, 24px);
+$ion-scale-700: var(--ion-scale-700, 28px);
+$ion-scale-800: var(--ion-scale-800, 32px);
+$ion-scale-900: var(--ion-scale-900, 36px);
+$ion-scale-1000: var(--ion-scale-1000, 40px);
+$ion-scale-1100: var(--ion-scale-1100, 44px);
+$ion-scale-1200: var(--ion-scale-1200, 48px);
+$ion-scale-1400: var(--ion-scale-1400, 56px);
+$ion-scale-1600: var(--ion-scale-1600, 64px);
+$ion-scale-1800: var(--ion-scale-1800, 72px);
+$ion-scale-2000: var(--ion-scale-2000, 80px);
+$ion-scale-2400: var(--ion-scale-2400, 96px);
+$ion-scale-2800: var(--ion-scale-2800, 112px);
+$ion-scale-3200: var(--ion-scale-3200, 128px);
+$ion-scale-3400: var(--ion-scale-3400, 136px);
+$ion-scale-3600: var(--ion-scale-3600, 144px);
+$ion-scale-4000: var(--ion-scale-4000, 160px);
+$ion-scale-5000: var(--ion-scale-5000, 200px);
+$ion-scale-6200: var(--ion-scale-6200, 248px);
+$ion-scale-7400: var(--ion-scale-7400, 296px);
+$ion-scale-9000: var(--ion-scale-9000, 360px);
+$ion-scale-025: var(--ion-scale-025, 1px);
+$ion-scale-050: var(--ion-scale-050, 2px);
+$ion-scale-075: var(--ion-scale-075, 3px);
+$ion-border-default: var(--ion-border-default, $ion-primitives-neutral-400);
+$ion-border-default-rgb: var(--ion-border-default-rgb, 213, 213, 213);
```

```suggestion
$ion-border-default-rgb: var(--ion-border-default-rgb, $ion-primitives-neutral-400-rgb);
```

Should this fallback to `$ion-primitives-neutral-400-rgb` (and the same for all other border rgb vars)?